src/main/java/com/shima/todo_list/UpdateRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.shima.todo_list;
+
+import java.time.LocalDate;
+
+public class UpdateRequest {
+
+    private LocalDate ScheduledEndDate;
+    private LocalDate ActualEndDate;
```

Comment:
@simamumu89 
ここ頭文字が大文字になってるのはあまりよろしくないですね！
```suggestion
    private LocalDate scheduledEndDate;
    private LocalDate actualEndDate;
```